services/movie.py

```diff
@@ -0,0 +1,45 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> QuerySet:
+    movies = Movie.objects.all()
+
+    if not genres_ids and not actors_ids:
+        return movies
+
+    if genres_ids and actors_ids:
+        movies = movies.filter(genres__id__in=genres_ids)
+        movies = movies.filter(actors__id__in=actors_ids)
```

Comment:
The current filtering logic uses an 'AND' condition when both `genres_ids` and `actors_ids` are provided. If the intention is to match any of the provided genres or actors, consider using `Q` objects to apply an 'OR' condition.